src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/NotYetReady/CountRows.txt

```diff
@@ -0,0 +1,332 @@
+// ******** TABLE WITH NO BLANK RECORDS ********
+
+>> CountRows([])
+0
+
+>> CountRows([0])
+1
+
+>> CountRows([1])
+1
+
+// NUMERIC RECORDS
+
+>> CountRows([1, 2, 3, 4, 5])
+5
+
+>> CountRows(Table({a:1},{a:2},{a:3}))
+3
+
+// STRING RECORDS
+
+>> CountRows(["1", "one", "two", "three", "four"])
+5
+
+>> CountRows(Table({a:"1"},{a:"two"},{a:"three"},{a:"four"},{a:"five"}))
+5
+
+// BOOLEAN RECORDS
+
+>> CountRows([true, false])
+2
+
+>> CountRows(Table({a:true},{a:false}))
+2
+
+// DATE/TIME/DATETIMEVALUE RECORDS
+
+>> CountRows([Date(2022,12,12), DateValue("10/01/2014")])
+2
+
+>> CountRows(Table({a:Date(2022,12,12)},{a:DateValue("10/01/2014")}))
+2
+
+>> CountRows([DateTimeValue("5/12/2022 6:30:30 PM"), DateTimeValue("May 19, 2022")])
+2
+
+>> CountRows(Table({a:DateTimeValue("5/12/2022 6:30:30 PM")},{a:DateTimeValue("May 19, 2022")}))
+2
+
+//  ALL DATATPYE RECORDS 
+
+﻿>> CountRows([1, "2", "two", true, false, Date(2022,12,12),Time(6,30,30),DateTimeValue("5/12/2022 6:30:30 PM")])
+8
+
+// ******** TABLE WITH NO BLANK RECORDS ********
+
+﻿>> CountRows([0,""])
+2
+
+>> CountRows([1,2,3,4,Blank()])
+5
+
+﻿>> CountRows([1, "2", true, false, Blank(), "", " ", Date(2022,12,12),Time(6,30,30),DateTimeValue("5/12/2022 6:30:30 PM")])
+10
+
+>> CountRows([1,"2", true, Blank(),Blank(),Blank()])
+6
+
+>> CountRows([1,Date(2022,12,12),"3",If(1<0,4),Blank()])
+5
+
+>> CountRows(Table({a:false},{a:Blank()},{a:true},{a:Blank()}))
+4
+
+>> CountRows(Table({a:"1"},{a:"2"},If(1<0,{a:"3"}),If(1>0,{a:"4"}),{a:"5"}))
+5
+
+>> CountRows(LastN([1,2,3,4], Blank()))
+Blank()

```

Comment:
The result should be 0

Reply:
@CarlosFigueiraMSFT  This expression is equating to CountRows(Blank()) which returns Blank() as per current behavior.
Do you want me to file a bug to address this issue?

![CountRowsBlank](https://user-images.githubusercontent.com/104027403/172249312-d7b652e4-d653-40d8-99ec-522c24afbd5b.jpg)

Reply:
In Canvas it currently returns blank, but it will change to return 0. The tests in PowerFx should be forward looking.

Reply:
Updated the expected result as 0

Reply:
{If 1 is less than [] then
Run D#} ¿